tests/controllers/user.test.js

```diff
@@ -102,7 +102,7 @@ test("user unsubscribe GET request with valid token returns error", async () =>
   const validToken = "0e2cb147-2041-4e5b-8ca9-494e773b2cf0";
 
   // Set up mocks
-  const req = { fluentFormat: jest.fn(), query: { token: validToken } };
+  const req = { fluentFormat: jest.fn(), query: { token: validToken, hash: "testHash" } };
```

Comment:
Maybe pedantic, but should we make the hash `ad9c69bcc69b3399775d2ddbe9b0b229369fca42` (i.e., sha1 of "testHash") so it's slightly more clear to someone reading the test that the hash value is a sha1 hash?